crl-janitor/pubsub.tf

```diff
@@ -0,0 +1,43 @@
+resource "google_pubsub_topic" "crl-janitor-pubsub-topic" {
+  count = var.enable ? 1 : 0
+
+
```

Comment:
optional nit: 2 newlines seems like a lot to separate the count

Reply:
I followed all other count style. Seems they all have 2 newlines

Reply:
Mm not sure what you referenced but I agree with Will, one newline is the way to go